test/scripts/filters/meta_generator.js

```diff
@@ -1,11 +1,19 @@
 'use strict';
 
+const decache = require('decache');
+
 describe('Meta Generator', () => {
   const Hexo = require('../../../lib/hexo');
   const hexo = new Hexo();
-  const metaGenerator = require('../../../lib/plugins/filter/after_render/meta_generator').bind(hexo);
+  let metaGenerator;
+  const metaGeneratorHelper = require('../../../lib/plugins/helper/meta_generator')(hexo);
   const cheerio = require('cheerio');
 
+  beforeEach(async () => {
+    await decache('../../../lib/plugins/filter/after_render/meta_generator');
```

Comment:
I don't think `await` has any effect here. I can't find any promise/async in decache's codebase.